tests/sccache_cargo.rs

```diff
@@ -302,24 +302,11 @@ fn test_gcp_arg_check() -> Result<()> {
     let mut cmd = Command::new(SCCACHE_BIN.as_os_str());
     cmd.arg("--start-server")
         .env("SCCACHE_LOG", "debug")
-        .env("SCCACHE_GCS_OAUTH_URL", "http://foobar");
+        .env("SCCACHE_GCS_OAUTH_URL", "http://127.0.0.1");
 
     cmd.assert().failure().stderr(predicate::str::contains(
         "If setting GCS credentials, SCCACHE_GCS_BUCKET",
     ));
 
-    stop_sccache()?;
```

Comment:
why did you remove it?

Reply:
maybe we should keep/transform it to have an invalid endpoint to test the error ?

Reply:
Nice catch. I found current logic may break the server spawn logic. Trying to fix.